### PR TITLE
add offline docs container

### PR DESCRIPTION
**Pull request by otan**

### CODE REVIEW

**Comment by keith-mcclellan** [comment]
stashing this somewhere will be good if we ever need to do this again (we probably will)

File: OFFLINE-DOCS.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Offline docs instructions
```
Comment (jlinder) [inline OFFLINE-DOCS.md:1]: This file only mentions the `./Dockerfile` file. What about the contents of `offline-build`?

File: offline-build/builder [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+image=cockroachdb/offline-docs
```
Comment (jlinder) [inline offline-build/builder:5]: Does this docker hub repo exist already?

File: OFFLINE-DOCS.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+* Either:
+  * Build the container: `docker build -t offline-docs .`
+  * Pull the container: `docker pull offline-docs`.
```
Comment (jlinder) [inline OFFLINE-DOCS.md:5]: Should this read:

```suggestion
   * Pull the container: `docker pull cockroachdb/offline-docs`. 
```

**Review by jlinder — COMMENTED** [review]
Why is this docker files at `Dockerfile` and  `offline-build/Dockerfile` appear to be the same. What is the intended difference?

File: OFFLINE-DOCS.md (hunk 3 above)
Comment (otan) [reply OFFLINE-DOCS.md:5]: Done!

File: OFFLINE-DOCS.md (hunk 1 above)
Comment (otan) [reply OFFLINE-DOCS.md:1]: I've deleted the root Dockerfile

File: offline-build/builder (hunk 2 above)
Comment (otan) [reply offline-build/builder:5]: No -- I think I need you to create it.
Also, not sure about deployment process for this yet. This is copy-pasta from elsewhere.

**Comment by otan** [comment]
duplicate items have been deleted :)

**Comment by otan** [comment]
guess we have no takers :P

**Pull request closed without merging**